code/game/objects/items.dm

```diff
@@ -630,3 +637,30 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 /obj/item/proc/microwave_act(obj/machinery/microwave/M)
 	if(M && M.dirty < 100)
 		M.dirty++
+
+/obj/item/proc/friendly_force()
+	if(force)
+		if(force <= 3)
+			f_force = "very low"
+		else if(force <= 5)
+			f_force = "low"
+		else if(force <= 7)
+			f_force = "medium"
+		else if(force <= 10)
+			f_force = "<font color=green>high</font>"
+		else if(force <= 15)
+			f_force = "<font color=red>Combat Grade</combat>"
+		else if(force <= 24)
+			f_force = "<font color=orange>high Combat Grade</font>"

```

Comment:
can you make these not autism, please? I like exceptionally lethal so do something more like that and less steelpointy